runtime/calamari/src/lib.rs

```diff
@@ -705,7 +698,6 @@ construct_runtime!(
 		// Handy utilities.
 		Utility: pallet_utility::{Pallet, Call, Event} = 40,
 		Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 41,
-		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>} = 42,
```

Comment:
Should we leave this commented, to not reuse the index ? Not sure if that would be a problem

Reply:
Just left a comment here.